tdrs-backend/docker-compose.yml

```diff
@@ -39,3 +49,6 @@ services:
       - "8080:8080"
     depends_on:
       - postgres
+      - localstack
+volumes:
+  localstack_data:
```

Comment:
This addition creates a [Docker Named Volume](https://spin.atomicobject.com/2019/07/11/docker-volumes-explained/) which will exist outside of the project directory and persist data between container restarts and removal.

[Additional Reference](https://nickjanetakis.com/blog/docker-tip-28-named-volumes-vs-path-based-volumes)